readthedocs/proxito/middleware.py

```diff
@@ -367,11 +369,33 @@ def _get_https_redirect(self, request):
 
         return None
 
+    def add_resolver_headers(self, request, response):
+        if hasattr(request, "unresolved_url") and request.unresolved_url is not None:
```

Comment:
This should be initialized as None, similar to

https://github.com/readthedocs/readthedocs.org/blob/ea4337133c6d5c07260aad819541fbb9afc3ca24/readthedocs/proxito/middleware.py#L199-L200